com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -676,13 +676,16 @@ def parallel(self, width, routing=Routing.ROUND_ROBIN, func=None):
             func: Optional function called when :py:const:`Routing.HASH_PARTITIONED` routing is specified.
                 The function provides an integer value to be used as the hash that determines
                 the tuple channel routing.
+            name (String): The name to display for the parallel region.
 
         Returns:
             Stream: A stream for which subsequent transformations will be executed in parallel.
 
         """
+        if name is None:
+            name = "parallel"
```

Comment:
Here's what it looks like when the stream name is used for the default parallel region name:

![newname2](https://user-images.githubusercontent.com/4752570/27928746-4019d694-6245-11e7-848f-0dc8e778211a.JPG)

As you can see, the `map_lambda` operator carries the name into the parallel region.